translations/translations.json

```diff
@@ -7671,8 +7671,8 @@
       "hostInformation": "Údaje o nabízejícím",
       "contactInformation": "Kontaktní údaje",
       "shelterButton": "Zobrazit",
-      "bed": "1 postel",
-      "beds": "{{beds}} postele",
+      "bed_one": "1 postel",
+      "bed_other": "{{beds}} postele",
```

Comment:
~~This has to be `{{count}}` actually, could you please change all occurrences?
https://www.i18next.com/translation-function/plurals~~

Edit: Nvm, we'll remove the shelter constants anyway.